### PR TITLE
Added Squadcast to the list of Alertmanager webhook receiver

### PR DESCRIPTION
**Pull request by raghuchinnannan**
Added Squadcast to the list of Alertmanager webhook receiver. Squadcast is an on-call alerting tool which helps you get alerted via Phone call, SMS, Email and Push notifications and lets you take actions on those alerts.
Signed-off-by: Raghu <raghu@squadcast.com>

### CODE REVIEW

File: content/docs/operating/integrations.md [hunk 1]
```diff
@@ -78,6 +78,7 @@ For notification mechanisms not natively supported by the Alertmanager, the
   * [Signal](https://github.com/dgl/alertmanager-webhook-signald)
   * [SMS](https://github.com/messagebird/sachet): supports [multiple providers](https://github.com/messagebird/sachet/blob/master/examples/config.yaml)
   * [SNMP traps](https://github.com/maxwo/snmp_notifier)
+  * [Squadcast](https://support.squadcast.com/docs/prometheus): helps you get alerted via Phone call, SMS, Email and Push notifications and lets you take actions on those alerts.
```
Comment (brian-brazil) [inline content/docs/operating/integrations.md:81]: Please remove the text, just the link is enough.

**Comment by brian-brazil** [comment]
Can you add the DCO? You might find it easiest to squash.

File: content/docs/operating/integrations.md (hunk 1 above)
Comment (raghuchinnannan) [reply content/docs/operating/integrations.md]: Updated and added DCO. Could you please approve?

**Comment by raghuchinnannan** [comment]
@brian-brazil Added the DCO. Thanks for the tip.

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**